### ISSUE TITLE
Prettier configuration not applied

### ISSUE THREAD

**Issue by idahogurl** [0x2aff/vscode-prettier-eslint#1]
1. Create the following files

`.prettierrc`
```json
{
    "semi": false,
    "singleQuote": false
}
```

`test.js`
```js
/* eslint-disable */
const options = {
  first: 'hello, world',
  second: true,
  internationalizing: true,
};

export { options };
```
2. Run "Format Document With..." command
3. Choose "VSCode Prettier ESLint"

**Expected**: File formatted with double quotes and no semi-colon
**Actual**: File formatted with single quotes and with semi-colon

**Comment by exceptionptr** [comment]
I cant confirm this.

The only thing I can confirm is when using .cjs Prettier config you have to reload VSCode to before the changes apply.
(Solving this requires restarting the workers which results in slow formats ...)

But using .prettierrc in JSON format works for me.

Do you get any output in your Output Buffer?

**Comment by idahogurl** [comment]
No, I did not get any output in the **Output** panel. Here is a link to the repository with all the files. https://github.com/idahogurl/prettierrc-example

**Comment by exceptionptr** [comment]
@idahogurl My bad ...

I haven't added `prettier-eslint` as a dependency in the README. I aim to keep the extension as slim as possible without bundling `prettier-eslint` into it.

I've updated the README and released a new version: v1.1.2. This version will now throw an error if the extension is unable to resolve `prettier-eslint`.  Just install the package, reload VSCode, and hopefully, it should work.

Could you test this, please? Thank you.

**Comment by idahogurl** [comment]
It is now working. Going to close this issue.
